recipes/xorg-xauth/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set xorg_name = "xauth" %}
+{% set xorg_category = "app" %}
+{% set name = "xorg-" ~ xorg_name %}
+{% set version = "1.1.2" %}
+{% set sha256 = "84d27a1023d8da524c134f424b312e53cb96e08871f96868aa20316bfcbbc054" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage(name|lower) }}
+
+requirements:
+  build:
+    - m2-autoconf  # [win]
+    - m2-automake{{ am_version }}  # [win]
+    - m2-libtool  # [win]
+    - m2w64-pkg-config  # [win]
+    - make
+    - pkg-config  # [not win]
+    - posix  # [win]
+    - gnuconfig  # [unix]
+    - {{ compiler('c') }}        # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - make                          # [unix]
+    - autoconf                      # [unix]
+    - automake                      # [unix]
+    - gettext                       # [unix]
+    - libtool                       # [unix]
+  host:
+    - pthread-stubs
+    - xorg-libx11 >=1.6,<2
+    # Additional constraintless xorg-libx11 dep below is for conda-build to add potential pinning to.
+    - xorg-libx11
+    - xorg-libxau
+    - xorg-libxext
+    - xorg-libxmu
+    - xorg-xtrans
+    - xorg-util-macros
+  run:
+    - {{ pin_compatible('xorg-libx11') }}
+    - {{ pin_compatible('xorg-libxau') }}
+    - {{ pin_compatible('xorg-libxext') }}
+    - {{ pin_compatible('xorg-libxmu') }}
```

Comment:
These have a `run_exports` and should be added automatically.
```suggestion
```

Reply:
Done.

Reply:
ERROR (xorg-xauth,bin/xauth): Needed DSO lib/libXmuu.so.1 found in ['conda-forge/linux-64::xorg-libxmu==1.1.3=h7f98852_0']
  ERROR (xorg-xauth,bin/xauth): .. but ['conda-forge/linux-64::xorg-libxmu==1.1.3=h7f98852_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)

@xhochy It seems that libxmu is an exception to this.

Reply:
Seems like this package has not been updated for quite a while. Can you please send a PR to it to fix it?